### DISCUSSION TITLE
Welcome to parametric_cgx_model Discussions!

### DISCUSSION THREAD

**Discussion by ostodieck** [daptablade/parametric_cgx_model#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Wraxq1020** [comment]
Hi I'm an Automation and Mechanical Engineer. 

I was wondering which version/ installation method you used to install calculix on windows. It doesn't seem easy to find 2.20 for windows.

**Reply by ostodieck** [reply]
Hi @Wraxq1020 - thanks for your question. 
The official binaries released on [](http://www.dhondt.de/) are for Linux, so I am actually running cgx and ccx through windows system for linux (WSL). This works for me so far... see installation commands I use below. If you want to use CGX with a graphical interface, you will also have to install an X-server like VcXsrv.  
Alternatively, have a look at the calculix forum [](https://calculix.discourse.group/)?
Hope this helps?

```
# install prerequisites (as required)
apt-get update && apt-get install -y --no-install-recommends \
        wget \
        bzip2 \
        libglu1 \
        libxi6 \
        libxmu6 \
        libgfortran4 \
  && rm -rf /var/lib/apt/lists/*

# install cgx
wget http://www.dhondt.de/cgx_2.20.bz2
bunzip2 cgx_2.20.bz2
mv cgx_2.20 /usr/local/bin/cgx
chmod ao+rx /usr/local/bin/cgx

# install ccx
wget http://www.dhondt.de/ccx_2.20.tar.bz2
bunzip2 ccx_2.20.tar.bz2 && tar -xvf ccx_2.20.tar
mv /usr/local/CalculiX/ccx_2.20/src/ccx_2.20 /usr/local/bin/ccx
chmod ao+rx /usr/local/bin/ccx
rm /usr/local/ccx_2.20.tar && rm -rf /usr/local/CalculiX
```
